config/manager/manager.yaml

```diff
@@ -42,7 +42,9 @@ spec:
             fieldRef:
               fieldPath: metadata.namespace
         - name: ADDON_NAME
+          value: sed_replace_addon_name
         - name: SOP_ENDPOINT
+          value: sed_replace_sop_endpoint
       - name: readiness-server
```

Comment:
This file is not part of the deploy script or tool, it is part of the general config bundle, we should not touch it just to fix something in the deploy script

Reply:
Good point, I'll have the deploy script add the values to the fields.